src/lib/actor.js

```diff
@@ -0,0 +1,19 @@
+export default class Actor{
+    constructor(ctx, stage){
+        this.ctx = ctx;
+        this.stage = stage;
+    }
+
+    render = (deltaTime) => {
+
+    }
+
+    update = (deltaTime) => {
+        
+    }
+
+    destroy = () => {
+        delete Actor.ctx;
+        delete Actor.stage;
```

Comment:
First off I'm assuming you meant:
```
delete this.ctx;
delete this.stage;
```

Secondly, we probably don't want to do this because deleting the reference doesn't do anything.
We should use ctx to remove the shape of the actor and then use the built in stage method
to remove this actor.

```suggestion
//clear boundingbox
ctx.clearRect(this.x, this.y, this.width, this.height);

//remove from stage
this.stage.removeActor(this);
```